gpt.py

```diff
@@ -18,7 +20,10 @@
 
 torch.manual_seed(1337)
 
-# wget https://raw.githubusercontent.com/karpathy/char-rnn/master/data/tinyshakespeare/input.txt
+if not os.path.exists('input.txt'): 
+    output = subprocess.check_output(["wget", "https://raw.githubusercontent.com/karpathy/char-rnn/master/data/tinyshakespeare/input.txt"])
+    print(output.decode())
+    
```

Comment:
I believe something similar should be done for this as well https://github.com/karpathy/ng-video-lecture/blob/381c613da07c82ad27c8b06a01f93583318571b0/bigram.py#L17-L19

Reply:
added to bigram.py as well.